scripts/asm-installer/install_asm

```diff
@@ -539,6 +559,8 @@ FLAGS:
                                       IAM permissions
      --enable_gcp_components          Allow the script to enable required GCP
                                       managed services and components
+     --enable_registration            Allow the script to register the cluster
+                                      as an environ memeber
```

Comment:
I think "register a cluster to an environ" is a bit clearer

Reply:
Done.